exec/java-exec/src/main/java/org/apache/drill/exec/physical/impl/aggregate/HashAggBatch.java

```diff
@@ -86,6 +99,67 @@
           "aggrValuesContainer" /* workspace container */, UPDATE_AGGR_INSIDE, UPDATE_AGGR_OUTSIDE, UPDATE_AGGR_INSIDE);
 
 
+  public int getOutputRowCount() {
+    return hashAggMemoryManager.getOutputRowCount();
+  }
+
+  public RecordBatchMemoryManager getRecordBatchMemoryManager() {
+    return hashAggMemoryManager;
+  }
+
+  private class HashAggMemoryManager extends RecordBatchMemoryManager {
```

Comment:
Can we pull this out into a separate file and add some unit tests for it?

Reply:
@ilooner I want to keep it consistent with the way I did for other operators.  Can I file a JIRA and do that later for all operators ? This code gets tested from tests written in testOutputBatchSize.java.